test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -483,33 +492,56 @@ def verify_option_exists(self, option:str):
             title = get_obj(CommunitySettingsComponents.MEMBERS_BUTTON.value).title
             verify_text(option, str(title))
         elif option=="Mint Tokens":
-            title = get_obj(CommunitySettingsComponents.MEMBERS_BUTTON.value).title
+            title = get_obj(CommunitySettingsComponents.MINT_TOKENS_BUTTON.value).title
             verify_text(option, str(title))         
               
     def select_community_settings_option(self, option:str):
         if option=="Permissions":
             click_obj_by_name(CommunitySettingsComponents.PERMISSIONS_BUTTON.value)
+        elif option=="Mint Tokens":
+            click_obj_by_name(CommunitySettingsComponents.MINT_TOKENS_BUTTON.value)
      
-    def verify_permission_screen_title(self, option:str):
+    def verify_welcome_screen_title(self, option:str):
         if option=="Permissions":
             title = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_TITLE.value).text
             verify_text(option, str(title))
+        elif option=="Tokens":
+            title = get_obj(CommunityTokensComponents.TOKENS_SCREEN_TITLE.value).text
+            verify_text(option, str(title))
              
-    def verify_welcome_permission_image(self):
-        path = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_IMAGE.value).source.path
-        verify_text_contains(str(path), "permissions2_3")
+    def verify_welcome_image(self, option:str):
```

Comment:
Here we can just get the object at first instance and only compare the `path`value in the if/else statements:

```suggestion
    def verify_welcome_image(self, option:str):
        path = get_obj(CommunityWelcomePageComponents.WELCOME_SCREEN_IMAGE.value).source.path        
        
        if option == "Permissions":
            verify_text_contains(str(path), "permissions2_3")

        elif option == "Tokens":
            verify_text_contains(str(path), "mint2_1")
```